esterni/piano-di-qualifica/components/valutazioni-miglioramento.tex

```diff
@@ -0,0 +1,56 @@
+
+  \rowcolors{2}{lightgray}{white!80!lightgray!100}
+  \begin{longtable}[H]{>{\centering\bfseries}m{4cm} >{\centering\arraybackslash}m{6cm} >{\centering\arraybackslash}m{6cm}}
+    \rowcolor{darkgray!90!}
+    \color{white}{\textbf{Dominio}} & \color{white}{\textbf{Problema}} & \color{white}{\textbf{Soluzione}} \\
+    Incontri di gruppo & Durante il periodo di progettazione il gruppo non ha potuto incontrarsi di persona a causa dell'epidemia di COVID-19.  & Grazie a strumenti di videoconferenza e videochiamata abbiamo potuto continuare le nostre attività. I nostri incontri verranno eseguiti sempre in modalità telematica fino al termine del progetto didattico o delle misure ristrettive.\\
+    Technology baseline & A causa di una mancata solida preparazione alla presentazione del Proof of Concept il gruppo si è dilungato in discorsi non richiesti, sforando il tempo a disposizione. & Il gruppo ha deciso di evitare questi eventi cercando di aumentare il tempo di preparazione alle varie presentazioni. \\
+  \end{longtable}
+
+
+\newpage
+  \subsection{Valutazioni sui ruoli}
+
+  \rowcolors{2}{lightgray}{white!80!lightgray!100}
+  \begin{longtable}[H]{>{\centering\bfseries}m{4cm} >{\centering\arraybackslash}m{6cm} >{\centering\arraybackslash}m{6cm}}
+    \rowcolor{darkgray!90!}
+    \color{white}{\textbf{Ruolo}} & \color{white}{\textbf{Problema}} & \color{white}{\textbf{Soluzione}} \\
+    Responsabile & Sono sorte diverse difficoltà da parte del responsabile nella suddivisione bilanciata delle ore tra i membri provocando sovraccarichi di lavoro. & Il gruppo ha deciso di incrementare l'attenzione nella suddivisione della mole di lavoro ai membri del gruppo. \\
+    Analista & Data l'inesperienza nell'analisi, sono sorti diversi problemi nell'individuazione dei requisiti e dei casi d'uso, causando rallentamenti. & La soluzione intrapresa è stata quella di contattare il proponente cercando quindi di velocizzare l'individuazione dei requisiti, in modo da non ritardare la consegna in RR\@. \\
+    Responsabile & A causa di una disattenzione, è stato sbagliato il preventivo di progetto, andando sotto la soglia richiesta. & Il gruppo ha deciso di attuare delle attività di verifica più consistenti, in modo da rilevare il prima possibile errori. \\
+    Verificatore & La poca esperienza nel campo, ha portato ai verificatori uno studio molto consistente dei documenti su cui attuare le verifiche, provocando ritardi, oppure mancate correzioni, come quella sopra menzionata. & Il gruppo ha deciso di far prestare molta più attenzione ai verificatori, attuando una politica di ispezione attraverso una check-list. \\
+    Programmatore & Sono sorte molte difficoltà nell'utilizzo delle tecnologie per colpa dell'inesperienza nel loro uso. Allungando di molto il tempo di autoapprendimento, ha portato il gruppo a saltare la seconda revisione. & Il gruppo ha deciso di sfruttare meglio il tempo a disposizione cercando di massimizzare le ore produttive. \\
```

Comment:
Anche qui la soluzione proposta è un po' alla buona intenzioni, secondo me è meglio evitare di dire che cerchiamo di fare qualcosa, e affermare invece cosa abbiamo istanziato per fare fronte al problema rilevato. In questo caso, ad esempio, andrebbe ristrutturata la frase proprio. (tipo come è scritto il punto seguente)